package.json

```diff
@@ -26,21 +26,21 @@
     "@actions/core": "^1.4.0",
     "@actions/exec": "^1.1.0",
     "@actions/github": "^5.0.0",
-    "@types/jest": "^26.0.23",
-    "@zeit/ncc": "^0.22.3",
     "fs": "0.0.1-security",
-    "jest": "^26.6.3",
-    "jest-junit": "^12.2.0",
-    "request": "^2.88.2",
-    "ts-jest": "^26.5.6",
-    "typescript": "^4.3.2",
-    "yarn": "^1.22.10"
+    "request": "^2.88.2"
```

Comment:
This could be removed. Could you validate that it isn’t used anywhere in the codebase? I believe it’s unused.

Reply:
Can confirm, it's unused now